### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Sindhu983**
Home page

### CODE REVIEW

File: home-page.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+.main-container{
+    position: sticky;
+    top: 0;
+    z-index: 1;
+}
+.title {
+    font-size: 2rem;
+    font-weight: 600;
+    margin-left: 1rem;
+    margin-top: 1.2rem;
```
Comment (VishalPatil18) [inline home-page.css:23]: You could use the single-line shorthand to achieve the same thing. 
e.g.
`margin: 1.2rem initial initial 1rem`
i.e.
`margin: [top] [right] [bottom] [left]`

File: home-page.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    color: rgb(4, 3, 8);
+
```
Comment (VishalPatil18) [inline home-page.css:25]: You could avoid blank lines in class properties.

File: home-page.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <script src="https://kit.fontawesome.com/8ddd15368f.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <link rel="stylesheet" href="/home-page.css">
+    <link rel="stylesheet" href="filter.css">
+    <link rel="stylesheet" href="/wishlist/wish-list.css">
```
Comment (AtulPant2704) [inline home-page.html:13]: Both of these stylesheet are not commited in this PR, you can remove both of these lines.

File: home-page.css [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+}
+.arrivals{
+    background-color: rgb(192, 190, 189);
+    padding: 30px;
+    text-align: center;
+}
+.card_contianer{
+    display: flex;
+}
```
Comment (VishalPatil18) [inline home-page.css:137]: You could add an empty line at the end of the file to avoid such warnings by Github

File: home-page.html [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+                        <a class="wishlist-btn btn" href="/wishlist/wish-list.html"><i class="fa fa-heart-o"></i>
+                        <span class="badge-wishlist">4</span></a>
+                    </li>
+                    <li><a class="add-cart-btn btn" href="/cart/cart.html"><i class="fa fa-shopping-cart"></i>
+                    <span class="badge-wishlist">2</span></a></li>
+                </ul>
+            </nav>
+        </header>
+    </div>
+    <img class="main-img" src="girl-image.jpg" alt="">
```
Comment (VishalPatil18) [inline home-page.html:38]: You could give value to the `alt` attribute as that helps for accessibility

**Review by AtulPant2704 — APPROVED** [review]

**Review by VishalPatil18 — APPROVED** [review]
A few suggestions were added, the rest looks quite good. Great work @Sindhu983 👍

**Pull request merged**